include/WCSimDetectorConstruction.hh

```diff
@@ -639,6 +639,7 @@ private:
   G4double vessel_radius_curv;                        // radius of the sphere to determine curvature of cap of pressure vessel
   G4double vessel_radius;                             // radius of the pressure vessel (spherical cap)
   G4double dist_pmt_vessel;                           // distance between glass of pmt and inner radius of pressure vessel (region where water/gel lives)
+  G4double pmtModuleHeight;                           // includes puck and single PMT support, not PMT base. The height of pmt module for solid works design
```

Comment:
make this variable available in `WCSimConstructPMT` and `WCSimConstructMultiPMT`